<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1031</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1031</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1031</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="tourBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTableView" name="placesViewTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>341</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="tourPhoto">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>131</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="tourName">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="uselessGuideLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Гид:</string>
        </property>
       </widget>
       <widget class="QLabel" name="guideLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="guidePhoneViewLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>телефон</string>
        </property>
       </widget>
       <widget class="QLabel" name="tourStartDateView">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Даты:</string>
        </property>
       </widget>
       <widget class="QLabel" name="tourStartViewLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>хх.хх.хх - хх.хх.хх</string>
        </property>
       </widget>
       <widget class="QLabel" name="placePhotoViewLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>370</y>
          <width>131</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>380</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>400</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>420</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Отель</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>440</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Даты:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>450</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>хх.хх.хх - хх.хх.хх</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tourTable"/>
     </item>
     <item>
      <widget class="QGroupBox" name="addBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>339</width>
         <height>489</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTableView" name="addPlacesTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>341</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="tourNameLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя тура</string>
        </property>
       </widget>
       <widget class="QLabel" name="guideNameLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя гида</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>450</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить тур</string>
        </property>
       </widget>
       <widget class="QLabel" name="guidePhoneLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Номер гида</string>
        </property>
       </widget>
       <widget class="QLabel" name="startDateLabel">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата начала</string>
        </property>
       </widget>
       <widget class="QLabel" name="expDateLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата окончания</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tourNameEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="guideNameEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="guidePhoneEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="placesLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Места посещения</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="placeNameEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="placeNameLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
       <widget class="QLabel" name="hotelLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Отель</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hotelNameEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="addressLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="addressNameEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tourPhotoButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Фото</string>
        </property>
       </widget>
       <widget class="QLabel" name="tourPhotoLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Файл не выбран</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="tourStartDate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="tourExpDate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="placePhotoButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Фото</string>
        </property>
       </widget>
       <widget class="QLabel" name="placePhotoLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Файл не выбран</string>
        </property>
       </widget>
       <widget class="QLabel" name="expDateLabel_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>290</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата окончания</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="placeStartDate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>270</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="placeExpDate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>290</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="startDateLabel_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата начала</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить место</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
